src/components/logout/logout_container.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
```

Comment:
That's what im saying perhaps each one of the components that are in a folder should have an index.js file to accomplish exactly the same thing, don't you think?